### PR TITLE
Improve code examples exprience

### LINKED ISSUE #1980: Improve syntax highlight for JSX in code examples
# Feature Request

## Description
Currently, our code examples syntax highlighter doesn't do a good job with JSX code. We should update this so JSX code is easier to read in the code examples.

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Resolves #1980

* Set language on syntax highlight to JSX to improve highlighting
* Improved syntax highlighter theme - based on ReactJS.org syntax highlighter theme
* Added aria-hidden to code examples to hide code from screen readers when closed

### Screenshot of new syntax theme
screen shot 2018-10-20 at 5 45 21 am

### CODE REVIEW

File: packages/terra-doc-template/src/syntax-highlighter-theme.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
+const prismColors = {
```
Comment (bjankord) [inline packages/terra-doc-template/src/syntax-highlighter-theme.js]: This file is based on https://github.com/reactjs/reactjs.org/blob/master/src/prism-styles.js

Comment (yuderekyu) [reply packages/terra-doc-template/src/syntax-highlighter-theme.js]: want to add a comment that references that?

Comment (yuderekyu) [reply packages/terra-doc-template/src/syntax-highlighter-theme.js]: want to add a comment that references that?

Comment (bjankord) [reply packages/terra-doc-template/src/syntax-highlighter-theme.js]: Updated: https://github.com/cerner/terra-core/pull/1981/commits/5a0f163094ab4f15c45e48c6a99ae96ce9c37f1b

**Pull request merged**